applications/Chat/evaluate/README.md

```diff
@@ -1,182 +1,217 @@
+| Precision<br/> Recall<br/> F1 Score | Measure the number of overlaps between prediction and reference (design for classification and extraction categories)                                                                               |
+
+## Evaluation Process
+### Data Format
+#### Target Answers / Predictions
+A JSON file contains one list. Each element in the list is a target answer / prediction record for one instruction / question.
+An element should have the following fields:
+
+* `category` (str, compulsory): The category of the instruction / question.
+* `instruction` (str, compulsory): The instruction / question for the LLM.
+* `input` (str, optional): The additional context of the instruction / question.
+* `output` (str, optional): The sample output of the instruction (default: GPT-3.5).
+* `target` (str, optional): The target answer for the instruction.
+* `id` (int, compulsory): The ID of the instruction / question.
+
+If the `input` has a target answer, the `output` can be empty. Otherwise, we generate answers from GPT-3.5/GPT-4 as the `output`, and the `target` field is empty.
+
+Example:
+```
+[
+    {
+        "category": "brainstorming",
+        "instruction": "请介绍一下人工智能的多个领域。",
+        "input": "",
+        "output": "{GPT-3.5 Answers}",
+        "target": "",
+        "id": 1
+    },
+    {
+        "category": "classification",
+        "instruction": "新闻标题：为什么电影《倩女幽魂》中燕赤霞一个道士却拿着金刚经？请根据新闻标题判断新闻所属的分类，你需要从文化，娱乐，体育，财经，房产，教育，科技，旅游，游戏，军事这十类中选择一个答案。",
+        "input": "",
+        "output": "",
+        "target": "{target answer}",
+        "id": 2
+    }
+]
+```
+
+#### Model Answers / Predictions
+
+A JSON file contains one list. Each element in the list is a model answer / prediction record for one instruction / question.
+
+An element should have the following fields:
+
+* `category` (str, compulsory): The category of the instruction / question.
+* `instruction` (str, compulsory): The instruction / question for the LLM.
+* `input` (str, optional): The additional context of the instruction / question.
+* `output` (str, compulsory): The output from the LLM.
+* `target` (str, optional): The target answer for the instruction.
+* `id` (int, compulsory): The ID of the instruction / question.
+
+Example:
+```
+[
+    {
+        "category": "brainstorming",
+        "instruction": "请介绍一下人工智能的多个领域。",
+        "input": "",
+        "output": "{GPT-3.5 Answers}",
+        "target": "{Model Answers / Predictions}",
+        "id": 1
+    },
+    {
+        "category": "classification",
+        "instruction": "新闻标题：为什么电影《倩女幽魂》中燕赤霞一个道士却拿着金刚经？请根据新闻标题判断新闻所属的分类，你需要从文化，娱乐，体育，财经，房产，教育，科技，旅游，游戏，军事这十类中选择一个答案。",
+        "input": "",
+        "output": "{Model Answers / Predictions}",
+        "target": "{target answer}",
+        "id": 2
```

Comment:
The `"output"` in **line 141** should be "`{Model Answers / Predictions}"`, and the `"target"` should be empty, I make mistakes here.
Since it is **"Model Answers / Predictions"** file, the `"target"` is still the`target answer`, `GPT-3.5 Answers` only be included in **"Target Answers / Predictions"** file as`"output"`.
The `"target"`  in **"Model Answers / Predictions"** file is not used in our current version of codes.